### PR TITLE
Dev build

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by tacaswell** [comment]
The actions won't run until we merge this I believe.

**Comment by tacaswell** [comment]
https://github.com/tacaswell/governance/runs/4946816339?check_suite_focus=true shows the workflow running.

**Comment by tacaswell** [comment]
While this is mostly mechanical, it does make some formatting changes to governance.md.  I propose we skip the full formality for changing the governance and only get 2 SC members to sign off.

**Review by efiring — APPROVED** [review]
Based on quick reading, it looks fine to me.

**Review by timhoffm — APPROVED** [review]

File: conf.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "N2SN Docs"
```
[the remark this answers is not in the document]
Comment (tacaswell) [reply conf.py:21]: Jargon from BNL where I templated this config file from 🤦🏻

**Review by timhoffm — APPROVED** [review]

**Comment by tacaswell** [comment]
https://matplotlib.org/governance/ <- finally live!

**Pull request merged**